docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -0,0 +1,116 @@
+
+:FeatureName: Support for analyzing .NET applications
+
+include::../snippets/developer-preview-admonition.adoc[]
+
+
+
+.Support for languages other than Java
+
+In {ProductFullName} 7.1.0, you can use {ProductShortName} to analyze .NET applications written in languages other than Java. To run analysis on .NET applications written in languages other than Java, add a custom rule set and do not specify a target language.
+
+:FeatureName: Support for languages other than Java
+
+include::../snippets/developer-preview-admonition.adoc[]
+
+
+
+.Assessment and review statuses are now displayed in the {ProductShortName} UI
+
+In earlier releases of {ProductShortName}, the status of the assessment and the review was only displayed for applications. With this update, the status of the archetype assessment and review processes is displayed in the {ProductFullName} user interface (UI):
+
+* The assessment statuses are the following:
+** Completed: All required assessments completed.
+** InProgress: The assessment process is in progress.
+** NotStarted: The assessment process has not been started.
+* The review statuses are the following:
+** Completed: A review exists.
+** NotStarted: The review process has not been started.
+
+
+
+.New Insights feature has been added
+
+Tagging rules that earlier generated tags and showed the presence of technology also generate insights now and show the location of code. While Insights do not impact the migration, they contain useful information about the technologies used in the application and their usage in the code. Insights do not have an effort and category assigned but might have a message and tag. You can view Insights in the Static report under the Insights tab.
+
+:FeatureName: Insights
+
+include::../snippets/technology-preview-admonition.adoc[]
+
+
+
+.New {ProductShortName} CLI options are available to select language providers when analyzing multi-language applications
+
+Previously, you could not select specific language providers to run separately for the analysis of a multi-language application. With this update, you can use the new `--provider` {ProductShortName} CLI option to explicitly set which language provider to run.
+
+The following CLI options are also available with this update:
+
+* `--list-providers` to list language providers supported for the analysis.
+* `--override-provider-settings` to override an existing supported language provider or to run your own unsupported provider.
+
+NOTE: You can now also configure supported language provider options in a provider’s configuration file.
+
+
+
+.A new Task Manager page is now available in the {ProductShortName} UI
+
+In earlier releases of {ProductShortName}, tasks that were being performed and the queue of pending tasks were not displayed in the {ProductFullName} user interface (UI). With this update, a new Task Manager page is available to view the following information about the tasks that are queued:
```

Comment:
In earlier releases of {ProductShortName}, tasks that were being performed and the queue of pending tasks were not displayed in the **{ProductShortName}** user interface (UI)

Reply:
Added suggestion, I prefer first using the full name and then a short name